### PR TITLE
add incubator projects

### PR DESCRIPTION
**Pull request by larsrh**
Finally, just in time for the Philadelphia Summit.

* typelevel/general#1
* typelevel/general#2
* typelevel/general#3
* typelevel/general#4
* typelevel/general#6
* typelevel/general#8

### CODE REVIEW

**Comment by larsrh** [comment]
Rendered version:
screenshot_20160227_210015

**Comment by adelbertc** [comment]
Could you change the blurb for Frameless since Spark no longer calls it Spark SQL? Perhaps something like "Expressive types for Spark"

**Comment by larsrh** [comment]
@adelbertc Done, see 323d715.

**Comment by adelbertc** [comment]
:+1: Thanks!

File: _data/incubator.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  description: "Purely functional basic blocks atop of Finagle for building composable HTTP APIs"
+  github: "https://github.com/finagle/finch"
+- title: "monix"
+  description: "High-performance asynchronous and event-based programming"
+  github: "https://github.com/monixio/monix"
+- title: "Frameless"
+  description: "Expressive types for Spark"
+  github: "https://github.com/adelbertc/frameless"
+- title: "eff-cats"
+  description: "Extendsible effects for Cats"
```
Comment (etorreborre) [inline _data/incubator.yml]: There is a small typo here: `s/Extendsible/Extensible`.

**Comment by alexandru** [comment]
:+1:

**Comment by milessabin** [comment]
:+1:

**Comment by milessabin** [comment]
Oops ... sorry ... I missed @etorreborre's comment.

**Comment by milessabin** [comment]
I'll fix that now.

**Pull request merged**